### ISSUE TITLE
Make tab-panels focusable

### ISSUE THREAD

**Issue by LucyGwilliamAdmin** [open-sdg/open-sdg#924]
**URL:** http://sdgdev-813006012.eu-west-1.elb.amazonaws.com/dac2020/1-2-1/

Tab-panels are not focusable and screen reader users are not able to navigate from the tabs using cursor keys, so in order to read the content within the tab it is necessary to tab to the next focusable element on the page, and then use the cursor keys to navigate back up the page.

**Current Code Ref(s):** #chartview

```
<div role="tabpanel" class="tabPanel" id="chartview" aria-hidden="false" aria-labelledby="tab-chartview" style="">[…]</div>
```

**Solution:**
Include the tabpanel in the page sequence when it is triggered so that it can be focused by screen reader users.

**Example:** 

```
<div role="tabpanel" class="tabPanel" id="chartview" aria-hidden="false" aria-labelledby="tab-chartview" style="" tabindex="0">[…]</div>
```
